projects/packages/forms/src/blocks/dropzone/index.js

```diff
@@ -0,0 +1,95 @@
+import { Path } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { getIconColor } from '../contact-form/util/block-icons';
+import renderMaterialIcon from '../contact-form/util/render-material-icon';
+import edit from './edit';
+import save from './save';
+
+const name = 'dropzone';
+const settings = {
+	apiVersion: 3,
+	title: __( 'File upload dropzone', 'jetpack-forms' ),
```

Comment:
Should this be title case?

```suggestion
	title: __( 'File Upload Dropzone', 'jetpack-forms' ),
```